mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -969,6 +971,14 @@ func ResourceComposerEnvironment() *schema.Resource {
 													ValidateFunc: validation.FloatAtLeast(0),
 													Description:  `Storage (GB) request and limit for DAG processor.`,
 												},
+												"count": {
+													Type:         schema.TypeInt,
+													Optional:     true,
+													ForceNew:     false,
+													Computed:     true,
+													ValidateFunc: validation.IntAtLeast(0),
```

Comment:
There is also upper limit (max 3) - is there a way to add it easily?

Reply:
Added upper limit.